docs/examples/compute/extended_metadata/extended_metadata.tf

```diff
@@ -0,0 +1,96 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+
+variable "compartment_ocid" {}
+variable "ssh_public_key" {}
+variable "ssh_private_key" {}
+
+variable "SubnetOCID" {}
+
+# Choose an Availability Domain
+variable "AD" {
+    default = "1"
+}
+
+variable "InstanceShape" {
+    default = "VM.Standard1.2"
+}
+
+variable "InstanceOS" {
+    default = "Oracle Linux"
+}
+
+variable "InstanceOSVersion" {
+    default = "7.3"
+}
```

Comment:
I've been simplifying new examples to put the above image related vars inside the baremetal_core_images datasource